mobile/mysterium/wireguard_connection_setup.go

```diff
@@ -259,6 +270,7 @@ func (w *wireguardDeviceImpl) applyConfig(devApi *device.Device, privateKey stri
 			// All traffic through this peer (unfortunately 0.0.0.0/0 didn't work as it was treated as ipv6)
 			AllowedIPs: []string{"0.0.0.0/1", "128.0.0.0/1"},
 		},
+		ReplacePeers: true,
```

Comment:
What does this do?

Reply:
It removed the old configuration from the old peers, and replace it with a new configuration. Instead of just adding over and over.